source/pervasive/multiset.rs

```diff
@@ -63,7 +63,20 @@ impl<V> Multiset<V> {
     /// Creates a multiset whose elements are given by the domain of the map `m` and whose 
     /// multiplicities are given by the corresponding values of `m[element]`. The map `m` 
     /// must be finite, or else this multiset is arbitrary.
-    pub spec fn new(m: Map<V, nat>) -> Self;
+    /// TODO(travis): no, that can't be right. If m is infinite, an
+    /// axiom below says it's finite, and now we get false?
```

Comment:
I don't believe that it's possible to prove false from the definitions and axioms.
I added two tests https://github.com/verus-lang/verus/commit/271e1a963b91660024b8a7fa3ac2f9e02060c9ef

Reply:
@utaal recorded this concern as 271e1a963b91660024b8a7fa3ac2f9e02060c9ef